packages/e2e-tests/src/specs/editor/shopping/testUtils.js

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
Why is this file here? Why not in e2e-test-utils?

Reply:
I'll move it --- was looking at another directory which has it in `testUtils.js` not sure which one it was (doesn't matter). 

<img width="884" alt="Screen Shot 2022-05-04 at 1 56 42 PM" src="https://user-images.githubusercontent.com/62242/166750590-f2e448e3-465d-422a-ab11-c9a3dba6afc5.png">

Reply:
We usually only have testUtils folders for JS unit tests or Karma tests.

Reply:
Why is this file still needed?

Reply:
🙃 forgot to remove -- just pushed an update